Source/Scene/Scene.js

```diff
@@ -1752,9 +1801,16 @@ define([
 
         this._tweens.update();
         this._camera.update(this._mode);
-        this._screenSpaceCameraController.update();
+
+        if (!this._useWebVR || this._mode === SceneMode.SCENE2D) {
```

Comment:
>Also, does it make sense to always disable the screen-space camera controller? Or is that a app decision? Won't it differ on desktop vs. mobile?

I think it does. Especially because we don't differentiate between the viewports. AFAIK, there really shouldn't be mouse input when using VR anyway. Try using the Occlus Rift with the Quake demo. Moving my head and using the mouse to look around make me nauseous. I have seen it when you have a virtual monitor in front of you and the mouse is used there. Using cardboard, there is no way to touch the screen.